packages/components/src/components/data-list/data-list.scss

```diff
@@ -93,6 +93,25 @@
     }
 }
 
+.statement__row--detail-text-mobile {
+    width: calc(100vw - 32px);
+    height: 100%;
+    text-align: left;
+    padding: 6px 16px;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    line-height: 1.43;
+    font-size: 14px;
```

Comment:
you can add font size to the `Text` element. 
example: <Text size='xs'>...</Text>